pom.xml

```diff
@@ -689,8 +689,8 @@
                 <module>apache-httpclient-2</module>
                 <module>apache-httpclient4</module>
                 <module>apache-httpclient</module>
-                <module>apache-kafka-2</module>
                 <module>apache-kafka</module>
+                <module>apache-kafka-2</module>
```

Comment:
Is this change necessary for the topic at hand? Generally, we should avoid changes of this nature and leave them for the editorial team to clean up at a later date.

Reply:
It's not reverted.